dtfs-central-api/src/v1/validation/utilisation-report-service/utilisation-report-validator.test.js

```diff
@@ -0,0 +1,99 @@
+const {
+  validateMonth, validateYear, validateFilePath, validateUtilisationReportData
+} = require('./utilisation-report-validator');
+
+describe('utilisation-report-validator', () => {
+  describe('validateMonth', () => {
+    it('returns null when a correct month is provided', async () => {
+      const validationError = validateMonth(1);
+
+      expect(validationError).toEqual(null);
+    });
+
+    it('returns an error when no month is provided', async () => {
+      const validationError = validateMonth(undefined);
+
+      expect(validationError).toEqual('Month is required');
+    });
+
+    it('returns null when an incorrect month is provided', async () => {
```

Comment:
I think all the 3rd tests should be named `returns an error...`
Wondering if we can commonise the tests too?

Reply:
I'll have a think about this one, might tackle it in FN-978 if that's alright!